roles/openshift_logging/files/generate-jks.sh

```diff
@@ -73,6 +73,7 @@ function generate_JKS_chain() {
         -keystore $dir/$NODE_NAME.jks \
         -storepass $ks_pass \
         -noprompt \
+        -trustcacerts \
```

Comment:
Im interested in understanding why you believe this resolves the issue.  If I read [1] and [2], it looks to me that this switch enables trust of the systemwide CAs that are installed on the runtime.  This means it will work IFF the cert you provide can be additionally trusted by one of the global CAs

[1] http://docs.oracle.com/javase/6/docs/technotes/tools/solaris/keytool.html
[2] http://docs.oracle.com/javase/6/docs/technotes/tools/solaris/keytool.html#cacerts